src/containers/ListCompanies/container.js

```diff
@@ -0,0 +1,9 @@
+import { connect } from 'react-redux';
+import ListCompanies from 'components/ListCompanies';
+
+const mapStateToProps = state => ({
+    companies: state.showCompanies.companies
+});
```

Comment:
Since `ListCompanies` is just presenting, I removed dispatch and the addCompany function and created the AddCompanies container.